src/main/py/commission/artfragment.py

```diff
@@ -14,9 +14,9 @@
 class ArtFragment:
     """
     Class to create ArtFragment
-    - artwork (Artwork): Artwork that ArtFragment is contributing to
-    - contributor_id (str): ID of peer generating the ArtFragment
-    - pixels (set[Pixel]): set of pixels that fragment occupies
+    - artwork_id (int): ID of Artwork that fragment is contributing to.
+    - contributor_pkey (int):
+    - pixels (set[Pixel]): set of pixels that fragment occupies.
```

Comment:
I think we should remove the types in the doc comments as changes are made and rely instead on type hints where helpful instead. They tend to get out of date and we already have several mismatches.